drivers/SmartThings/zigbee-switch/profiles/switch-color-temp.yml

```diff
@@ -0,0 +1,24 @@
+name: switch-color-temp
+components:
+- id: main
+  capabilities:
+  - id: switch
+    version: 1
+  - id: switchLevel
+    version: 1
+    config:
+      values:
+        - key: "level.value"
+          range: [1, 100]
+  - id: colorTemperature
+    version: 1
+    config:
+      values:
+        - key: "colorTemperature.value"
+          range: [ 2700, 5000 ]
+  - id: firmwareUpdate
+    version: 1
+  - id: refresh
+    version: 1
+  categories:
+  - name: Switch
```

Comment:
Is this meant to appear as a switch or a light bulb on the app? If it is similar to the `SAMSUNG-ITM-Z-001`, then this should have the `Light` category so it appears as a bulb on the app. There is already a profile that is the same as this one but it uses the `Light` category, so I would suggest using the `color-temp-bulb-2700K-5000K.yml` profile instead of creating this new one

Reply:
I want to it appears as a switch. Because it is a smart switch device which controls lighting. I fully agree with you that it has same profile contents as the color-temp-bulb-2700K-5000K.yml. But the category is different that not a light but a switch.

Reply:
Thank you for explaining, that makes sense to me. Could you rename the profile to `switch-color-temp-2700K-5000K.yml`? Change both the filename and the `name:` field. This will follow the naming pattern established by the `color-temp-bulb` profiles. Other than that, everything looks good!

Reply:
Thank you for your rapid service. I really appriciate it :) I chaged both of file name and the name field in file.